### PR TITLE
fix(structure): mock folder structure

### PR DESCRIPTION
**Pull request by safeamiiir**
- Rename  `ExperimentDataMock` into `ExperimentsDataMock` (plural)
- Fix typo `extention` into `extension` 
- Restructure `__mocks__`:
```
.
└── __mocks__
    ├── mutation
    │   ├── experiment
    │   └── extension
    └── queries
        ├── experiment
        ├── extension
        └── user
```

### CODE REVIEW

**Review by samiralavi — APPROVED** [review]

File: aqueductcore/frontend/src/__mocks__/AppContextAQDMock.tsx [hunk 1]
```diff
@@ -91,20 +99,23 @@ function AppContextAQDMock({
...
     getAllTags_mock[getAllTags_mockMockMode],
-    getExperimentFiles_mock[getExperimentFiles_mockMockMode],
     ...getAllExperiments_mock[getAllExperiments_mockMockMode],
     ...getAllExperimentsWithNameFilter_mock[getAllExperimentsWithNameFilter_mockMockMode],
     ...getAllExperimentsWithTagFilter_mock[getAllExperimentsWithTagFilter_mockMockMode],
     ...getAllExperimentsWithStartTime_mock[getAllExperimentsWithStartTime_mockMockMode],
     ...getAllExperimentsWithEndTime_mock[getAllExperimentsWithEndTime_mockMockMode],
     ...updateExperiment_mock[updateExperiment_mockMockMode],
     ...removeExperiment_mock[removeExperiment_mockMockMode],
+    getExperimentFiles_mock[getExperimentFiles_mockMockMode],
```
Comment (str-anger) [inline aqueductcore/frontend/src/__mocks__/AppContextAQDMock.tsx:111]: is this ok it doesn't have `...` in the beginning?

**Review by str-anger — APPROVED** [review]
Who am I to stop this beauty from merging? Also, one minor question.

File: aqueductcore/frontend/src/__mocks__/AppContextAQDMock.tsx (hunk 1 above)
Comment (safeamiiir) [reply aqueductcore/frontend/src/__mocks__/AppContextAQDMock.tsx:111]: Yeah some of them might not be an array and we don't spread them.

**Review by jatinriverlane — APPROVED** [review]
Thanks @safeamiiir

**Pull request merged**